docs/tutorials/index.rst

```diff
@@ -78,3 +78,11 @@ Prometheus Graphs
    :maxdepth: 1
 
    robusta-runner-metrics
+
+HolmesGPT
```

Comment:
Would it be better to put it in the AI integration menu, when you configure holmes?